packages/core/src/types/index.ts

```diff
@@ -46,76 +55,54 @@ export enum RefineStrategy {
 
 export enum TillingScheme {
   NONE = 0,
-  QUADTREE = 1,
-  OCTREE = 2
+  QUADTREE = 2,
+  OCTREE = 3
 }
 
 export enum TileState {
   LOADING = 1,
   VISIBLE = 2
 }
 
-export class Tileset<T extends Tile<T>> {
-  public root: T;
-  public state: Map<number, TileState>;
-
-  private queue: PriorityQueue<T>;
-
-  constructor(root: T) {
-    this.root = root;
-    this.state = new Map();
-    this.queue = new PriorityQueue(400);
-  }
-
-  public getTiles(camera: Camera, threshold: number): T[] {
-    const planes = Frustum.GetPlanes(camera.getTransformationMatrix());
-
-    this.queue.reset();
-    this.queue.insert(this.root.screenSpaceError(camera), this.root);
-
-    const result: T[] = [];
-
-    while (!this.queue.isEmpty()) {
-      const { score: error, data: tile } = this.queue.extractMax();
-      if (!tile) {
-        break;
-      }
-
-      // Determine visibility
-      if (!tile.boundingInfo.isInFrustum(planes)) {
-        continue;
-      }
-
-      // If error below threshold skip loading child nodes
-      if (error < threshold) {
-        // If tile is already loading skip
-        if (this.state.get(tile.id) === TileState.LOADING) {
-          continue;
-        }
-
-        result.push(tile);
-      } else {
-        // If refine stategy is `ADD` add current node
-        if (tile.refineStrategy === RefineStrategy.ADD) {
-          // If tile is already loading skip
-          if (this.state.get(tile.id) === TileState.LOADING) {
-            continue;
-          }
-
-          result.push(tile);
-        }
-
-        for (const child of tile.children) {
-          this.queue.insert(child.screenSpaceError(camera), child);
-        }
-      }
-    }
-
-    return result;
-  }
-}
-
 export type FrameDetails = {
   zoom: number;
   level: number;
 };
+
+export type SceneOptions = {
+  /**
+   * Global coordinate system of the scene. It will be inherited from the
+   * image layer if it exists.
+   */
+  crs?: string;
+
+  /**
+   * Global trnasformation from coordinate system to pixel space. It will be inherited from the
+   * image layer if it exists.
+   */
+  transformation?: Matrix;
+
+  /**
+   * The scene's bounding volume to be used for initializing cameras
+   */
+  extents: BoundingInfo;
+};
+
+export type GeometryDataContent = {
+  uri: string;
+  region: { dimension: string; min: number; max: number }[];
```

Comment:
can you add a commnt about what acceptable values of `dimension` nominally look like?